src/plugins/formulas/utils.js

```diff
@@ -1,63 +1,30 @@
-/**
- * Check if provided expression is valid formula expression.
- *
- * @param {*} expression Expression to check.
- * @returns {Boolean}
- */
-export function isFormulaExpression(expression) {
-  return typeof expression === 'string' && expression.length >= 2 && expression.charAt(0) === '=';
-}
+// TODO: docs
+export function parseHFValue(value) {
+  if (value && value.constructor && value.constructor.name.includes('Error')) {
+    return parseErrorObject(value);
 
-/**
- * Check if provided formula expression is escaped.
- *
- * @param {*} expression Expression to check.
- * @returns {Boolean}
- */
-export function isFormulaExpressionEscaped(expression) {
-  return typeof expression === 'string' && expression.charAt(0) === '\'' && expression.charAt(1) === '=';
-}
+  } else if (typeof value === 'symbol') { // TODO: parsing `Symbol`s received from HF to `null`, needs to be checked if `Symbol` always means `null`.
```

Comment:
Reference and compare with `EmptyValue` symbol:
https://github.com/handsontable/hyperformula/blob/e696d1d8226ca80d2f5c653771d52dd4857a573d/src/Cell.ts#L24

```suggestion
  } else if (value === HyperFormula.EmptyValue) {.
```